src/audio/kpb.c

```diff
@@ -105,8 +106,7 @@ static struct comp_dev *kpb_new(struct sof_ipc_comp *comp)
 	if (!dev)
 		return NULL;
 
-	assert(!memcpy_s(&dev->comp, sizeof(struct sof_ipc_comp_process),
-			 comp, sizeof(struct sof_ipc_comp_process)));
+	memcpy(&dev->comp, comp, sizeof(struct sof_ipc_comp_process));
```

Comment:
Is this the rebase mistake?
I know it doesn't make much sense with the same src and dst sizes, but I would prefer to use memcpy_s to stay consistent across the source code and with checkpatch.

Reply:
Yes a rebase mistake. Thanks for spotting it.